c_tests/tests/simple.c

```diff
@@ -69,10 +73,9 @@ int main(int argc, char **argv) {
     res += 2;
     i--;
   }
-  abort(); // FIXME: unreachable due to aborting guard failure earlier.
+  printf("exit");
```

Comment:
Is this print testing anything useful?

Reply:
Yes, it's so that we know if we are not executing it twice, once in the stopgap interpreter, and then again in the compiled interpreter. In other words, it checks that the new control point return design does what we think it does and exists properly if the stopgap interpreter has interpreted a return.

Reply:
Aha!